packages/cashier/src/stores/account-prompt-dialog-store.ts

```diff
@@ -1,18 +1,18 @@
 import { observable, action } from 'mobx';
 import { isCryptocurrency } from '@deriv/shared';
+import { TRootStore } from 'Types';
 
 export default class AccountPromptDialogStore {
-    constructor(root_store) {
-        this.root_store = root_store;
-    }
+    // eslint-disable-next-line no-useless-constructor
+    constructor(public root_store: TRootStore) {}
 
     @observable should_show = false;
     @observable is_confirmed = false;
-    @observable last_location = null;
-    @observable current_location = null;
+    @observable last_location = '';
```

Comment:
Should we create a nullable type? Something like `type Nullable<T> = T | null`

Reply:
[As we talked] Adding this may be like having `any` type in our package. So we will move as it is.